src/pages/MapPage.js

```diff
@@ -1,10 +1,15 @@
 import React, {  useCallback, useRef } from "react";
+import { useLocation } from 'react-router-dom';
 import { toJpeg } from 'html-to-image'
+
 import Map from "./Map"
```

Comment:
Inconsistent use of `'` or `"` in all string literals. Please use `'` only.

Reply:
thanks @jofftiquez. resolved.

i intentionally disabled prettier for this because my changes are affecting a lot of other lines.

btw, several files aren't formatted well. moving forward it would be nice for this to have have a uniform lint config.

Reply:
can you create an issue mentioning uniform lint config @gballesteros10 ?

Reply:
done @warengonzaga #23